lighthouse-core/aggregator/aggregate.js

```diff
@@ -67,24 +67,28 @@ class Aggregate {
    * @private
    * @param {!AuditResult} result The audit's output value.
    * @param {!AggregationCriterion} expected The aggregation's expected value and weighting for this result.
+   * @param {!string} name The name of the audit.
    * @return {number} The weighted result.
    */
-  static _convertToWeight(result, expected) {
+  static _convertToWeight(result, expected, name) {
     let weight = 0;
 
     if (typeof expected === 'undefined' ||
         typeof expected.rawValue === 'undefined' ||
         typeof expected.weight === 'undefined') {
-      return weight;
+      const msg =
+          `Config for ${name} is undefined or does not contain rawValue and weight properties`;
```

Comment:
maybe add 'aggregations: ' to the beginning of these strings to indicate where in the config this is referring to?